jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBQueryResultSet.java

```diff
@@ -76,12 +52,13 @@
   private List<String> columnTypeList; // no deduplication
   private Map<String, Integer> columnInfoMap; // used because the server returns deduplicated columns
   private List<String> columnTypeDeduplicatedList; // deduplicated from columnTypeList
-  private RowRecord record;
+  private int rowsIndex = 0;
   private int fetchSize;
   private boolean emptyResultSet = false;
 
   private TSQueryDataSet tsQueryDataSet = null;
-  private ByteBuffer byteBuffer = null;
+  private byte[][] record;
```

Comment:
add some javadoc for this field..

You could separate the time bytes from this record. This record could rename to values